api-docs/rst/dev-guide/concepts.rst

```diff
@@ -798,3 +808,31 @@ To get started with RackConnect v3 cloud bursting:
 
 .. _RackConnect product information: http://www.rackspace.com/cloud/hybrid/rackconnect
 .. _Knowledge Center articles for RackConnect: http://www.rackspace.com/knowledge_center/product-page/rackconnect
+
+
+.. _convergence-concept:
+
+Convergence
+~~~~~~~~~~~
+
+The convergence feature provides higher reliability for scaling by optimizing
+the use of the Cloud Servers API with retries until they are successful.
+Convergence ensures that the current server and load balancer configuration
+for a scaling group always matches the specification in the launch configuration
+of the group. It does this by continuously converging to the desired state of the
+scaling group, instead of manipulating servers only once.
+
+Convergence also provides a self-healing capability by tracking all the servers
+in autoscaling group continuously and automatically replacing any servers that
+have been deleted out-of-band or transitioned to an ``ERROR`` state.
+
+It is used internally to launch and delete servers. You can trigger it explicitly 
+by submitting a :ref:`converge <trigger-convergence>` request on a group. This
+operation is useful for fixing a scaling group that is in ``ERROR`` state. Typically,
+the ``ERROR`` state is caused by an invalid launch configuration, for example a
+configuration that includes a server image reference of a deleted image.
+After correcting the launch configuration, you can submit a convergence request
+to restore the group to the desired state.
+
+This has been released on December 8th 2015. Kindly check
+:ref:`release notes <document-release-notes/as_20151204>` for any behavioral changes.
```

Comment:
The convergence feature provides higher reliability for scaling by optimizing
the use of the Cloud Servers API with retries until they are successful.
Convergence ensures that the current server and load balancer configuration
for a scaling group always matches the specification in the launch configuration
of the group. It does this by continuously converging to the desired state of the
scaling group, instead of manipulating servers only once.

Convergence also provides a self-healing capability by tracking all the servers
in an autoscaling group continuously and automatically replacing any servers that
have been deleted out-of-band or transitioned to an ``ERROR`` state.

The Autoscale process uses convergence internally to launch and delete servers. You can trigger convergence explicitly by submitting a :ref:`converge <post-converge-v1.0-tenantid-groups-groupid>` request for a specified group. This
operation is useful for fixing a scaling group that is in an ``ERROR`` state. Typically,
the ``ERROR`` state is caused by an invalid launch configuration, for example a
configuration that includes a server image reference for a deleted image.
After correcting the launch configuration, you can submit a converge request
to restore the group to the desired state.

.. note:: 
    The convergence feature was released December 8th, 2015. See the release
    :ref:`release notes <document-release-notes/as_20151204>` for information about behavioral changes 
     associated with the release.


Manish-- I don't think you need to include this note.  People can read the release notes to learn about changes.  If you want to include it, I suggest modifying so it is similar to the version in this comment.